### DISCUSSION TITLE
Preview HTML

### DISCUSSION THREAD

**Discussion by tlianza** [coder/code-server#3128] [Q&A] [answered]
I've played with some extensions but they don't seem to be working on code-server... all I'd like to do is preview HTML. I don't have a node-app and don't need to run a special server... literally just browse an html file who's source I'm editing. Any ideas?

**Comment by jsjoeio** [comment]
This is a great question...I just spent some time googling and haven't seen anything that doesn't require a special server.

One idea that comes to mind:

1. Create a Markdown `.md` file
2. Add an HTML codeblock with ```html```
3. Open with Markdown Preview

**Comment by code-asher** [comment, marked as answer]
Another possible way is to browse static files using the `/static` endpoint.

https://github.com/cdr/code-server/issues/1292#issuecomment-702334047

**Reply by tlianza** [reply]
Yes! This works perfectly and is exactly what I wanted. Thank you!!

**Comment by fongfai** [comment]
docker run -p   the node start port
